CancerGov/_src/Scripts/NCI/UX/PageSpecific/Inner/InnerPage.scss

```diff
@@ -16,6 +16,7 @@
 @import "../../../../../StyleSheets/nokia-maps";
 @import "../../../../../StyleSheets/cts-listing";
 @import "../../../../../StyleSheets/directory-search";
+@import "../../../../../StyleSheets/related-links";
```

Comment:
Want to take the ~Stylesheets aliasing for a test run or save it all for later?

Reply:
It would be an easy RegEx search and replace. Which branch should it be done in though? Not this one. Perhaps [microsite support](https://github.com/NCIOCPL/wcms-front-end/tree/microsite-support) branch?

Reply:
Oh, I meant, just for this file.